.github/workflows/pre-main.yaml

```diff
@@ -55,6 +55,16 @@ jobs:
 
       - name: make vet
         run: make vet
+  
+  deprecated-k8s-check:
+    name: Pluto Deprecated Version Check
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Download Pluto
+        uses: FairwindsOps/pluto/github-action@master
+      - name: Use pluto
+        run: |
+          pluto detect-files
```

Comment:
Is it useful to run pluto in this repo, since we do not maintain yaml files or helm chart releases?

Reply:
Probably not.  I just figured it was a good place to test the integration.  I'll close.